app/src/main/res/layout/fragment_reading_lists.xml

```diff
@@ -61,15 +61,14 @@
                     style="@style/MaterialLargePrimaryTitle"
```

Comment:
Forgot to mention but should this title also be bolded?

Reply:
Done